SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -83,14 +86,21 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         return try? String.init(contentsOf: accessTokenURL, encoding: .utf8)
     }
     
-    func toggleUIState(isEnabled: Bool) {
+    func toggleUIState(isEnabled: Bool, showCallControl: Bool) {
         placeCallButton.isEnabled = isEnabled
+        if (showCallControl) {
+            callControlView.isHidden = false
+            muteSwitch.isOn = false
+            speakerSwitch.isOn = true
+        } else {
+            callControlView.isHidden = true
+        }
     }
```

Comment:
Suggestion : I would factor out line 90 to 97 into a new method `toggleCallUIState` so that we can call it from other places rather calling the IBAction from our code.